### PR TITLE
feat: Add support for showing confidence intervals

### LINKED ISSUE #1857: Development: Non-symetric error bar - visualization
**Description**
Implement support for asymmetric uncertainties in visualize based on the design. 

An example data set can be found here:  https://test.visualize.admin.ch/browse?previous=%7B%22includeDrafts%22%3Atrue%7D&dataset=https%3A%2F%2Fmock.ld.admin.ch%2Fcube%2Fmock-two-sided%2F1&dataSource=Test

**Dependencies**
#1856

### PR DESCRIPTION
**Pull request by bprusinowski**
Closes #1857

This PR adds support for displaying confidence intervals in Visualize.

**Details**
- Added `ConfidenceUpperBoundDimension` and `ConfidenceLowerBoundDimension`
- Updated Y field type for columns and lines to correctly indicate that it can contain additional, uncertainty-related properties
- Standardized error formatting logic across all charts
- Added a way to display confidence intervals, which use the existing `ErrorWhiskers` components and tooltip formatting
- Fixed showing errors in symbol layer tooltip

## How to test
1. Go to [this link](https://visualization-tool-git-feat-non-symmetrical-error-bar-ixt1.vercel.app/en/create/new?cube=https://mock.ld.admin.ch/cube/mock-two-sided/1&dataSource=Test).
2. ✅ See that confidence intervals are displayed by default.
3. ✅ Open Vertical Axis field and see that there's an option to control their appearance.
4. ✅ Hover over columns and see that the tooltip displays the values in a different way than standard errors.

### CODE REVIEW

File: app/graphql/resolvers/index.ts [hunk 1]
```diff
@@ -63,8 +63,21 @@ export const resolveDimensionType = (
   timeUnit: ResolvedDimension["data"]["timeUnit"] | undefined,
   related: ResolvedDimension["data"]["related"]
 ): DimensionType => {
+  const relatedTypes = Array.from(new Set(related.map((d) => d.type)));
+
+  if (relatedTypes.length > 1) {
+    console.warn(
+      `WARNING: dimension has more than 1 related type`,
```
Comment (noahonyejese) [inline app/graphql/resolvers/index.ts:70]: Should this information be communicated to the user?

File: app/locales/de/messages.po [hunk 2]
```diff
@@ -411,6 +411,7 @@ msgstr "Es ist mindestens eine numerische Massnahme erforderlich."
 msgid "controls.chart.disabled.same-unit"
 msgstr "Es sind mindestens zwei Zahlenwerte mit gleicher Einheit erforderlich."
 
+#: app/charts/index.ts
```
Comment (noahonyejese) [inline app/locales/de/messages.po:414]: I don't know why this is happening, but it also happened to me last time I used `:extract` or `:pull`

File: app/locales/en/messages.po [hunk 3]
```diff
@@ -411,6 +411,7 @@ msgstr "At least one numerical measure is required."
 msgid "controls.chart.disabled.same-unit"
 msgstr "At least two numerical measures with the same unit are required."
 
+#: app/charts/index.ts
```
Comment (noahonyejese) [inline app/locales/en/messages.po:414]: 🔎

File: app/locales/it/messages.po [hunk 4]
```diff
@@ -411,6 +411,7 @@ msgstr "È richiesta almeno una misura numerica."
 msgid "controls.chart.disabled.same-unit"
 msgstr "Sono necessarie almeno due misure numeriche con la stessa unità."
 
+#: app/charts/index.ts
```
Comment (noahonyejese) [inline app/locales/it/messages.po:414]: It does it for all langs

**Review by noahonyejese — APPROVED** [review]
LGTM, nice work 🎉

File: app/locales/de/messages.po (hunk 2 above)
Comment (bprusinowski) [reply app/locales/de/messages.po:414]: Yeah, I am also not sure – I removed it a few times, but it keeps getting back, so I left it in the end 😅 As it's partially an auto-generated file, I think it should be fine to leave it 👍

**Comment by bprusinowski** [comment]
Thanks for the review and feedback @sosiology. I improved this in https://github.com/visualize-admin/visualization-tool/pull/1915 👍

**Comment by sosiology** [comment]
Nice thanks for the adjustment in #1915 @bprusinowski - LGTM

**Pull request merged**